lib/facilities/vha_facility.rb

```diff
@@ -39,6 +40,17 @@ def add_mental_health(facilities)
         end
       end
 
+      def format_phone_numbers(facilities)
```

Comment:
This extension issue is present in other facility types and I haven't changed those files. I'm not a big fan of this fix, since it iterates through the entire facilities set again. What do you guys think of me combining the `add_mental_health`, `format_phone_numbers`, and `add_websites` methods? Or more specifically, moving their guts into methods and pulling out the map so we only iterate once?

Reply:
It seems like phone numbers would be handled the same way regardless of facility type. Would it be better to have phone number formatting on `BaseFacility` in that case? If that's what you're suggesting, then 👍

Reply:
And why does this iterate through the entire facilities set again? Would it be simpler to change the phone numbers before they're stored in the `base_facilities` table as each facility is being read in from GIS/ArcGIS?

Reply:
Moving the phone numbers formatting to the parser